### PR TITLE
Updated documentation about how to setup a service principal for Azure

### PR DESCRIPTION
**Pull request by whites11**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/getting-started/cloud-provider-accounts/azure/index.md [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
-  - /guides/prepare-azure-subscription-for-tenant-clusters/
-  - /guides/prepare-azure-subscription/
-owner:
-  - https://github.com/orgs/giantswarm/teams/team-celestial
 ---
 
 # Prepare an Azure subscription to run Giant Swarm workload clusters
 
-In a Giant Swarm installation the workload clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the management cluster. This gives greater flexibility depending on the requirements and the use case. For example, it allows the management cluster to be running in one Azure subscription, while workload clusters operate in different Azure subscriptions, depending on the customer entities using them.
+In a Giant Swarm installation the workload clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the control plane. This gives greater flexibility depending on the requirements and the use case. For example, it allows the control plane to be running in one Azure subscription, while workload clusters operate in different Azure subscriptions, depending on the customer entities using them.
```
Comment (tuommaki) [inline src/content/getting-started/cloud-provider-accounts/azure/index.md:15]: Why these changes? I think we intentionally moved from `control plane` -> `management cluster` because `control plane` term is ambiguous between MC and WC?

See https://github.com/giantswarm/giantswarm/issues/14961

Comment (whites11) [reply src/content/getting-started/cloud-provider-accounts/azure/index.md:15]: This is a rebase error, thanks for catching.
Let me fix it

File: src/content/guides/prepare-azure-subscription-for-a-new-installation/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+date: "2020-10-14"
+type: page
+weight: 100
+tags: ["tutorial"]
+---
+
+# Prepare an Azure subscription to run a Giant Swarm intallation
+
+In a Giant Swarm installation the software managing workload clusters (the kubernetes clusters that run your workloads)
+is installed in a separate Kubernetes cluster, called `control plane`.
```
Comment (fiunchinho) [inline src/content/guides/prepare-azure-subscription-for-a-new-installation/index.md:13]:
```suggestion
is installed in a separate Kubernetes cluster, called `management cluster`.
```

Comment (fiunchinho) [reply src/content/guides/prepare-azure-subscription-for-a-new-installation/index.md:13]: Not just here, but in other parts of the file.

**Review by fiunchinho — CHANGES_REQUESTED** [review]
use `management cluster` instead of `control-plane`

**Review by fiunchinho — APPROVED** [review]
